noether_tpp/include/noether_tpp/tool_path_planners/raster/direction_generators.h

```diff
@@ -12,6 +12,7 @@ class FixedDirectionGenerator : public DirectionGenerator
 public:
   FixedDirectionGenerator(const Eigen::Vector3d& direction);
   Eigen::Vector3d generate(const pcl::PolygonMesh& mesh) const override final;
+  std::unique_ptr<DirectionGenerator> clone() const override final;
```

Comment:
I thought we were maybe leaning away from `unique_ptr`? Too much headache, too little benefit?  I could be mistaken.

Reply:
This clone function is not actually needed, so I removed it